components/molecules/Table.js

```diff
@@ -0,0 +1,76 @@
+import { useContext } from "react";
+import { LanguageContext } from "../../context/languageProvider";
+import en from "../../locales/en";
+import fr from "../../locales/fr";
+
+function Table({ rows, columns }) {
+  return (
+    <table className="w-full cursor-text text-left">
+      <thead className="text-sm font-display">
+        <tr>
+          <th key={columns[0].heading} className="pl-6 pb-2">
+            {columns[0].heading}
+          </th>
+          <th key={columns[1].heading} className="text-center pb-2">
+            {columns[1].heading}
+          </th>
+        </tr>
+      </thead>
+      <tbody className="divide-y-2 divide-gray-light-100 border-2 border-gray-light-100">
+        {rows.map(({ text, subtext, link, href }, idx) => (
+          <tr key={`${text[idx]}`}>
+            <td className="pl-6 py-4 border-r-2 border-dk-blue">
+              <div className="text-h6 font-display">{text}</div>
+              <div className="text-sm font-body">{subtext}</div>
+            </td>
+            <td className="text-center bg-gray-light-200">
+              <a
+                href={href}
+                className="cursor-pointer hover:text-custom-blue-link hover:underline focus:text-custom-blue-link focus:underline"
+              >
+                {link}
+              </a>
+            </td>
+          </tr>
+        ))}
+      </tbody>
+    </table>
+  );
+}
+
+export default function TableData() {
+  const { items } = useContext(LanguageContext);
+  const language = items.language;
+  const t = language === "en" ? en : fr;
+
```

Comment:
Okay, so this is where the content is actually passed in?

Reply:
Yep, data gets passed from the locale files into the structures which are used to build the table